### ISSUE TITLE
Does termd is appropriate for making a ssh command proxy?

### ISSUE THREAD

**Issue by wcc526** [termd/termd#13]
Does termd is appropriate for making a ssh command proxy? 

such as
 
https://www.sshtools.com/

There is my scene .

user  ->  ssh proxy server  -> target ssh target server

user  connect to the target server by ssh protocol with the ssh proxy server.

user's command will deliver to the target server by the ssh proxy server, and the command will be logging for the security purpose.

I need to run java sshd proxy program, Does termd is appropriate for making a ssh command proxy?

**Comment by vietj** [comment]
do you mean something like https://www.sshtools.com/en/solutions/ssh-gateway ?

**Comment by wcc526** [comment]
yes, I need to run sshd program or something terminal emulator for the command deliver.

And the terminal emulator is run on the ssh proxy server,all commands input by the user will deliver to the target server,and the command will be audit at the same time.

**Comment by vietj** [comment]
I believe it could be used for doing that, using the PtyMaster, so the target terminal would run itself as a PTY.
